app/src/main/java/a/a/a/aB.java

```diff
@@ -23,18 +23,19 @@
  */
 public class aB extends Dialog {
 
-    private int dialogImageResId = -1;
-    private String dialogTitleText = "";
-    private String dialogMessageText = "";
-    private boolean msgIsSelectable = false;
-    private View dialogCustomView;
-    private View dialogButtonsContainer;
-    private String dialogDefaultText = "Default";
-    private View.OnClickListener dialogDefaultListener = null;
-    private String dialogNoText = "No";
-    private View.OnClickListener dialogNoListener = null;
-    private String dialogYesText = "Yes";
-    private View.OnClickListener dialogYesListener = null;
+    // I don't know if it's bad  make everything public, but I did this so I could use .aB.apply in kotlin
```

Comment:
Fuck fuck kotlin

Reply:
what

Reply:
They love java keep it java

Reply:
@SyntaxSpin First tell me where did I say that I will recreate aB in kotlin?

Reply:
Hasrat Said it idk

Reply:
nah, he said "fuck fuck kotlin"